src/main/webapp/app/modules/canvas/visualization/visualization-properties/partials/properties/properties.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { IRootState } from 'app/shared/reducers';
+import { connect } from 'react-redux';
+
+import { IFeature } from 'app/shared/model/feature.model';
+import { IVisualMetadataSet, Property } from 'app/shared/model/visualMetadata.model';
+import { Checkbox, Switch, Item, Picker, TextField } from '@adobe/react-spectrum';
+import { parseBool, parseString } from 'app/shared/util/common-utils.ts';
+
+export interface IPropertiesProps extends StateProps, DispatchProps {
+  features: readonly IFeature[];
+  visual: IVisualMetadataSet;
+  propstype: string;
+  property: Property;
+}
+
+const Properties = (props: IPropertiesProps) => {
+  const [property, setProperty] = useState([]);
+
+  const handleCheckboxChange = e => {
+    props.property.value = !props.property.value;
+    setProperty([props.property.value]);
+  };
+
+  const handleValueChange = (value, possibleValues = null) => {
+    if (possibleValues) {
+      const selectedValue = possibleValues.filter(item => {
+        return item.value === value;
+      });
+      props.property.value = selectedValue[0];
+    } else {
+      props.property.value = value;
+    }
+    setProperty([props.property.value]);
+  };
+
+  return (
+    <>
+      {props.property.type === 'NUMBER' && (
+        <TextField
+          type="NUMBER"
+          onChange={text => {
+            handleValueChange(text);
+          }}
+          value={props.property.value.toString() || ''}
```

Comment:
place default values in model of property and import from there

Reply:
every property have diffrerent values 
`export interface Property {
  type?: PropertyType;
  createdBy?: null;
  createdDate?: Date;
  lastModifiedBy?: null;
  lastModifiedDate?: Date;
  id?: null;
  propertyType?: PropertyTypePropertyType;
  order?: number;
  value?: boolean | ValueElement | Color | number | null | string;
}
`